projects/week_1/Instructions.md

```diff
@@ -0,0 +1,332 @@
+
+Now, apply both transformations to `tag_orientation`. This should look basically identical to how you applied the transformations to `position`.
+
+Finally, we need to convert our new orientation back into a quaternion message and set it in the `new_tag` message. Here again, we've got a helper function for you called `transformationMatrixToQuaternionMessage()`. Call this function, passing `tag_orientation`, and assign the result to `new_tag.pose.orientation`.
+
+And that's it! We've written all the code we need to make this node complete.
+
+### 2.9 Build project
+
+Before we can run our newly finished node, we need to build it. Open a terminal window and navigate to the training workspace folder (`/home/robojackets/training_ws`).
+
+Source the underlay setup script:
+
+```bash
+$ source /opt/ros/foxy/setup.bash
+```
+
+And use colcon to build the workspace
+
+```bash
+$ colcon build
+```
+
+If your build succeeded, the last line printed will be `Summary: __ packages finished [_.__s]` that tells you how many packages were built and how long the process took.
+
+If any errors were found during the build, the final summary will tell you, and the output will include the error details. The summary of a failed build looks like this:
+
+```bash
+Summary: 16 packages finished [5.12s]
+  1 package failed: coordinate_transform
+  1 package had stderr output: coordinate_transform
+```
+
+If you do have any errors, you'll need to fix them before you can run your node. We recommend starting with the first error reported, reading the error message carefully, fixing it, and building again. Repeat that process until all errors are resolved. Of course, if you're not sure how to fix any given error, reach out to the trainers.
+
+### 2.10 Run project
+
+To run this project, we'll need two terminal windows or tabs.
+
+In the first terminal, after sourcing the underlay setup file and the training workspace overlay setup file, launch the week 1 launch file.
+
+```bash
+source /opt/ros/foxy/setup.bash
+source ~/training_ws/install/setup.bash
+ros2 launch rj_training_bringup week_1.launch.xml
+```
+
+This will open up the simulator and a pre-configured rviz instance. A number of other nodes will also be started in the background, including the coordinate transform node you wrote!
+
+In the second terminal, we need to run a node that can drive the robot around. You can either use keyboard control or joystick control.
+
+For joystick control:
+
+```bash
+$ ros2 launch traini_bringup joystick_control.launch.py
+```
+
+Or, for keyboard control:
+
+```bash
+$ ros2 run teleop_twist_keyboard teleop_twist_keyboard
+```
+
+Now you should be able to drive the robot around and see the tag detections visualized in the rviz window.
+
+If everything's working correctly, you'll see something like this:
```

Comment:
Where the blue box is the location rviz thinks the tag should be and the green one is where your code placed it.